dev/breeze/src/airflow_breeze/commands/release_candidate_command.py

```diff
@@ -164,9 +164,9 @@ def push_artifacts_to_asf_repo(version, repo_root):
             check=True,
         )
         console_print("Files pushed to svn")
+        os.chdir(repo_root)
```

Comment:
I don't think you should need this change. `remove_old_releases` moves to a different dir anyways, then moves back to the repo root.

Reply:
At that point, we are in the "versions" directory. We can't change to "asf-dist..." right there because it cannot be found.

Reply:
I see. Can we the change `remove_old_releases` to do this "reset to repo root" instead?